Makefile

```diff
@@ -704,6 +704,23 @@ generate_yaml_coredump: export ENABLE_COREDUMP=true
 generate_yaml_coredump:
 	$(MAKE) generate_yaml
 
+# TODO(howardjohn) clean all of this up
+istio-auth-mcp.yaml:
```

Comment:
any reason not to add this section to the `generate_e2e_test_yaml` below and update the caller?

Reply:
No strong reason, no. Everything here is a huge mess right now so I wanted to make the simplest change I could right now so we can get these tests passing while I work on cleaning it up (https://github.com/istio/istio/issues/11961). I won't leave the todo there for long.